grpc/src/main/java/io/stargate/grpc/service/Service.java

```diff
@@ -426,7 +428,22 @@ private void executePrepared(
           .whenComplete(
               (result, t) -> {
                 if (t != null) {
-                  handleException(t, responseObserver);
+                  if (t instanceof PreparedQueryNotFoundException) {
```

Comment:
Are we handling those in other modules?

Reply:
Logic borrowed from persistence-api 
https://github.com/stargate/stargate/blob/6452957cd078f17f1c0067c943e3fc2129f61ceb/persistence-api/src/main/java/io/stargate/db/datastore/PersistenceBackedDataStore.java#L115